docs/content/components/prompts/yes-or-no-choice.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Yes or no choice
+titleTemplate: ":title · Prompts · Noora · Tuist"
+description: A component that prompts the user to answer a yes or no question.
+---
+
+# Yes or no choice
+
+This component is a simple component that prompts the user to answer a yes or no question. It is a good way to get a quick answer from the user.
+
+| Property | Value |
+| --- | --- |
+| Interactivity | Required (fails otherwise) |
```

Comment:
I believe this should also be removed.

Reply:
Not really. I'd keep it to make it sure that people understand this component is interactive and terminal interactivity is required for it to work. I adjusted the logic to abort in the case of misusages.

Reply:
ahh, ok, I misunderstood this table.